Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -405,7 +406,9 @@ def get_incident_by_id_command(client, args):
 
     result = client.http_request('GET', url_suffix)
     incident = incident_data_to_xsoar_format(result)
-    readable_output = tableToMarkdown(f'Incident {inc_id} details', incident,
+    content = incident.copy()
+    content['IncidentUrl'] = create_clickable_url(content.get('IncidentUrl'))
+    readable_output = tableToMarkdown(f'Incident {inc_id} details', content,
```

Comment:
i think you can use `url_keys` instead of `create_clickable_url`

Reply:
Right. Wasn't familiar with this. Updated!